modules/internal/ChapelBase.chpl

```diff
@@ -831,19 +831,37 @@ module ChapelBase {
     return ret;
   }
 
+  inline proc sizeof_raw(type t): size_t {
```

Comment:
Let's call this _ddata_sizeof_element and have it accept the _ddata type as an argument